less/events.less

```diff
@@ -40,7 +39,26 @@
     background-image: url("@{image-url}");
   }
 }
+/* Define the function to map the event type with the background images ends */
 
+/* Reusable classes */
+.margin-bottom-10 {
```

Comment:
By using the classes I mentioned earlier, you will be able to remove those ones from your code.